tests/devices_test/dependencies_test.py

```diff
@@ -129,3 +137,121 @@ def test_dependencies_handling(self):
+            self.fail("Improper handling of missing libblockdev plugin")
+        # TODO: remove line (workaround (2/2) for blivet.reset fail)
+        self.unload_all_plugins()
+
+        disk1 = self.bvt.devicetree.get_device_by_name("disk1")
+
+        with six.assertRaisesRegex(self, ValueError, "requires unavailable_dependencies"):
+            self.bvt.initialize_disk(disk1)
+
+        pv = self.bvt.new_partition(size=Size("8GiB"), fmt_type="lvmpv")
+        pv_fail = self.bvt.new_partition(size=Size("8GiB"), fmt_type="lvmpv")
+        btrfs = self.bvt.new_partition(size=Size("1GiB"), fmt_type="btrfs")
+        raid1 = self.bvt.new_partition(size=Size("1GiB"), fmt_type="mdmember")
+        raid2 = self.bvt.new_partition(size=Size("1GiB"), fmt_type="mdmember")
+
+        with six.assertRaisesRegex(self, ValueError, "resource to create this format.*unavailable"):
+            self.bvt.create_device(pv_fail)
+
+        self.load_all_plugins()
+        self.bvt.create_device(pv)
+        self.bvt.create_device(btrfs)
+        self.unload_all_plugins()
+
+        try:
+            vg = self.bvt.new_vg(parents=[pv])
+        except blockdev.BlockDevNotImplementedError:  # pylint: disable=catching-non-exception
+            self.fail("Improper handling of missing libblockdev plugin")
+
+        self.load_all_plugins()
+        self.bvt.create_device(vg)
+        self.unload_all_plugins()
+
+        try:
+            lv1 = self.bvt.new_lv(fmt_type="ext4", size=Size("1GiB"), parents=[vg])
+            lv2 = self.bvt.new_lv(fmt_type="ext4", size=Size("1GiB"), parents=[vg])
+            lv3 = self.bvt.new_lv(fmt_type="luks", size=Size("1GiB"), parents=[vg])
+        except blockdev.BlockDevNotImplementedError:  # pylint: disable=catching-non-exception
+            self.fail("Improper handling of missing libblockdev plugin")
+
+        self.load_all_plugins()
+        self.bvt.create_device(lv1)
+        self.bvt.create_device(lv2)
+        self.bvt.create_device(lv3)
+        self.unload_all_plugins()
+
+        try:
+            pool = self.bvt.new_lv_from_lvs(vg, name='pool', seg_type="thin-pool", from_lvs=[lv1, lv2])
+        except blockdev.BlockDevNotImplementedError:  # pylint: disable=catching-non-exception
+            self.fail("Improper handling of missing libblockdev plugin")
+
+        self.load_all_plugins()
+        self.bvt.create_device(pool)
+        self.unload_all_plugins()
+
+        with six.assertRaisesRegex(self, ValueError, "requires unavailable_dependencies"):
+            self.bvt.destroy_device(pool)
+
+        try:
+            self.bvt.new_btrfs(parents=[btrfs])
+        except blockdev.BlockDevNotImplementedError:  # pylint: disable=catching-non-exception
+            self.fail("Improper handling of missing libblockdev plugin")
+
+        with patch.object(StorageDevice, "resizable", new_callable=PropertyMock(return_value=True)):
```

Comment:
Is this patch still needed? Resizable should be set to false if we don't have libblockdev.

Reply:
Fixed.